### DISCUSSION TITLE
Revo Hemera XS

### DISCUSSION THREAD

**Discussion by MarcoGasparcnc** [mriscoc/Ender3V2S1#797] [Q&A]
Hello,

I'm using Ender3V2-422-BLTUBL-MPC-20230312.bin on stock ender 3 v2 Neo working 5* :D 

I buy a Revo Hemera XS what i have to do to use it with mriscoc firmware?

Best Regards,
Marco Gaspar

**Comment by mriscoc** [comment]
Check https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
